### ISSUE TITLE
logstash.conf hidden and (a lot)verbose via output.stdout

### ISSUE THREAD

**Issue by zeph** [elastic/helm-charts#499]
**Chart version: 7.6.0**

**Describe the bug:**
output.stdout plugin is enabled by default as ./docker/data/logstash/pipeline/default.conf gets copied over to pipeline/logstash.conf ... the result is a verbose (up to uncontrollable flooding) spitting out logstash image to stdout of every single value ingested

**Steps to reproduce:**

1. install helm logstash, push to it anything not on the standard beat 5044 port
2. if you follow the example in the values.yaml, you are not tempted to override logstash.conf
3. logstash.conf is there and undocumented, it will produce lots of garbage

**Expected behavior:**
a sane error handling

**Provide logs and/or server output (if relevant):**
better not

**Any additional context:**
yeah, at least the information in the values.yaml about the section: 
```
# Allows you to add any pipeline files in /usr/share/logstash/pipeline/
logstashPipeline: {}
#  uptime.conf: |
#    input {
#      exec {
#        command => "uptime"
#        interval => 30
#      }
#    }
#    output { stdout { } }
```

shall maybe use `logstash.conf` (giving a BIG hint to override it) instead of adding more pipelines

**Comment by jmlrt** [comment]
Fixed by #500

**Comment by BenDavidAaron** [comment]
How does one override the default logstash.conf?
